h2o-py/tests/testdir_algos/coxph/pyunit_coxph_concordance.py

```diff
@@ -20,8 +20,6 @@ def coxph_smoke():
 
     metrics = coxph.model_performance(heart)
     assert 0.581 > metrics.concordance() and 0.580 < metrics.concordance()
-    assert 3696 == metrics.concordant()
```

Comment:
intentional?

Reply:
Rebased, the test was redesigned in an other PR to use lifelines instead of hardcoded results.